packages/babel-plugin-syntax-import-meta/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@babel/plugin-syntax-import-meta",
+  "version": "7.0.0-beta.34",
+  "description": "Allow parsing of import.meta",
+  "repository": "https://github.com/babel/babel/tree/master/packages/babel-plugin-syntax-import-meta",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "keywords": [
+    "babel-plugin"
+  ],
+  "peerDependencies": {
+    "@babel/core": "7.0.0-beta.34"
```

Comment:
If you look at our other plugins, we've been doing a range for this dependency.

Reply:
I looked through _all_ babel-plugin-syntax packages and some others, it seems like they are using all the same value.

Though I checked the history, it appears that a range has been used prior to _7.0.0-beta.31_

Reply:
Oh dang, that may have been an accident. I'll have to look into it, but it won't be until next week. Guess if it is already like that this is fine then.